src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs

```diff
@@ -367,9 +367,9 @@ IEnumerable<Config> GetAotConfigs ()
 						}
 					}
 
-					libs.Add (Path.Combine (toolchainLibDir, "libgcc.a"));
-					libs.Add (Path.Combine (androidLibPath, "libc.so"));
-					libs.Add (Path.Combine (androidLibPath, "libm.so"));
+					libs.Add ($"\\\"{Path.Combine (toolchainLibDir, "libgcc.a")}\\\"");
```

Comment:
You can do 

```
@$"\"{foo}\""
```

it makes for nice code :)

Reply:
indeed :-)

Reply:
I had to revert it to the original escaping as I was getting compiler errors when used with `@$` or `$@` as well.

Reply:
I just tested this and it worked

```
int f = 1;
$"Foo \"{f}\"";
```

not sure why we need the extra `\\` bits.

Reply:
We need extra `\\` to end with `\"` in the response file.

Using `@$` didn't help in this case and I was getting compiler errors.
```
  Tasks/Aot.cs(370,16): error CS8401: To use '@$' instead of '$@' for an interpolated verbatim string, please use language version 8.0 or greater. [/Users/rodo/git/xa-master/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj]
  Tasks/Aot.cs(370,21): error CS1026: ) expected [/Users/rodo/git/xa-master/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj]
  Tasks/Aot.cs(370,21): error CS1002: ; expected [/Users/rodo/git/xa-master/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj]
  Tasks/Aot.cs(370,64): error CS1002: ; expected [/Users/rodo/git/xa-master/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj]
  Tasks/Aot.cs(370,65): error CS1056: Unexpected character '\' [/Users/rodo/git/xa-master/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj]
  Tasks/Aot.cs(370,68): error CS1002: ; expected [/Users/rodo/git/xa-master/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj]
  Tasks/Aot.cs(370,68): error CS1513: } expected [/Users/rodo/git/xa-master/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj]
```
and similar error with `$@`. `@` doesn't play very well with quotes. I guess I could use `$@"\""{...}\"""` which doesn't help much IMO.